### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by bikash0070** [Astronxd/NetTV-WorldLink#2]
bro can you suggest new method i have try but it does not work it just show ad only

**Comment by Astronxd** [comment]
Its showing ad only means that you failed to spoof mac address. Mention the device, android version, steps you followed for me to help you.

**Comment by bikash0070** [comment]
on base of luncher i am able to login into my account but getting ad only on case of live tv that means is there any way to spoof mac address for livetv

**Comment by Astronxd** [comment]
As I mentioned previously, for me to help you, I need few information:
1: NetTV device model number (better picture)
2: Device model number
3: Android version of your device
4: Steps you performed

**Comment by bikash0070** [comment]
S905 SoC White color box
Using emulator menu
7
install apk and used clone app to Mac change  and other steps and install it and delete original file

**Comment by Astronxd** [comment]
just a quick confirmation did you change your mac address to that provided by worldlink? also did you add semicolon (:) in mac address?

**Comment by bikash0070** [comment]
yes i did same as you told and it works for the launcher but case on live tv it will stock on ad

**Comment by Astronxd** [comment]
I have released apk for streamz+ device. it should work with that one.

Edit: Check release 1.1

**Comment by bikash0070** [comment]
thanks for apk but i am unable pass ad and also can you tell me on which device or emulator you have tested

**Comment by Astronxd** [comment]
Tested on 
Genymotion android 9.0
Oneplus 9 pro android 14
Oneplus 8T android 13
Oppo CPH2124 android 12
ViaTV box android 9.0

**Comment by Astronxd** [comment]
If you can post a video of you following all the steps, I might be able to help you.

**Comment by krishnastunning** [comment]
> If you can post a video of you following all the steps, I might be able to help you.

bro baru tappai tutun video banuna milxa ki
